### ISSUE TITLE
Short between 3.3V and SCL on some boards

### ISSUE THREAD

**Issue by mwelling** [oshwabadge2020/Badge-PCB#33]
2 of the 4 prototype units I was sent from Sparkfun had a short from 3.3V to SCL.

After inspecting the PCB layout, Alex noticed that the two nets are nearest under U5.

![image](https://user-images.githubusercontent.com/1570196/73626778-0a4cbe00-45ff-11ea-877d-46067aa88e95.png)

You will see that they directly adjacent and the solder mask is improperly specified for the pads allowing for easier shorts.

I recommend updating U5's pattern and possibly doing thermals on the power pour to avoid manufacturing issues. Couldn't hurt to check the paste apertures as well.

**Comment by pdp7** [comment]
Is this fixed with https://github.com/oshwabadge2020/Badge-PCB/commit/0f4dcff1dcd95b07f539cf87d5421f1a353eb1b8 from @acamilo ?

**Comment by pdp7** [comment]
@mwelling @acamilo Should I order new PCBs?

**Comment by pdp7** [comment]
@nseidle Would you be able to look at the PCBs you received an check if there is short between TP5 to TP7, and TP6 to TP7?

We think the footprint on U5 was wrong.  Both the boards I received have I2C bus shorted.

**Comment by mwelling** [comment]
We can leave this open until we find the source of the shorts.

**Comment by siddacious** [comment]
probably not useful but I can confirm that VCC/3V and SCL are bridged on my board. I'll pull U5 and order a breakout from work for testing/dev until I can get an updated board

**Comment by pdp7** [comment]
hopefully  this is resolved with the updated footprint for U5 on Rev1.

I just removed U5 from Rev0 so I could use i2c.

**Comment by siddacious** [comment]
@pdp7 @mwelling here's a picture of my U5 pads after removal:
![image](https://user-images.githubusercontent.com/1183985/74592714-e1ea9980-4fd8-11ea-8b53-d7c41c670ec7.png)
I'm near certain that this will be addressed by @acamilo 's fix

**Comment by pdp7** [comment]
@acamilo did U5 work on your Rev1?

**Comment by mwelling** [comment]
Yes it worked, he did the i2c scan yesterday and all of the devices appeared.
